alws/crud/build_node.py

```diff
@@ -134,11 +144,48 @@ async def __process_rpms(pulp_client: PulpClient, task_id: int, task_arch: str,
                     href=href
                 )
             )
+    if exceptions:
+        raise ArtifactConversionError(
+            'Cannot put RPM packages into Pulp storage: %s',
+            '\n'.join([str(e) for e in exceptions]))
+    for repo_href, packages in repo_packages_mapping.items():
+        try:
+            await pulp_client.modify_repository(repo_href, add=packages)
```

Comment:
We can make this requests in parallel, so whole thing will be much faster on large number of repositories

Reply:
As for autopublish: it creates several tasks per each package while this approach will only make 1 call for new repo version/publication after all packages added

Reply:
Also, each task has only a handful of repositories, basically 3: src, arch, debug. Not sure if parallelization here will make any difference